metric_learn/lmnn.py

```diff
@@ -48,6 +48,9 @@ def _process_inputs(self, X, labels):
     self.X = X
     self.L = np.eye(X.shape[1])
     required_k = np.bincount(self.label_inds).min()
+    if self.params['debug']:
+        # Readying the stats to store the objective function
+        self.stats = np.zeros(self.params['max_iter']-1)
```

Comment:
Test coverage is very important, especially for features like this which are off by default.